packages/ui/src/showcases/DepositAndBorrowPage.stories.tsx

```diff
@@ -0,0 +1,346 @@
+      walletBalance: '1.250.000$',
+      APY: '3.5%',
+    },
+    {
+      assets: 'BTC',
+      walletBalance: '1.250.000$',
+      APY: '3.5%',
+    },
+  ]
+  const DepositTitleBox = () => (
+    <StackLayout gap={2.125}>
+      <Text as="span" color="oxfordBlue" type="heading 2">
+        Your deposit
+      </Text>
+      <ColumnLayout center gap={1}>
+        <Text color="oxfordBlue" as="span" type="body 2">
+          Balance $
+          <Text as="span" color="oxfordBlue" bold>
+            1.00
+          </Text>
+        </Text>
+
+        <Text color="oxfordBlue" as="span" type="body 2">
+          Collateral $
+          <Text as="span" color="oxfordBlue" bold>
+            1.00
+          </Text>
+        </Text>
+      </ColumnLayout>
+    </StackLayout>
+  )
+
+  const BorrowTitleBox = () => (
+    <BorrowColumn>
+      <StackLayout gap={2.125}>
+        <Text as="span" color="oxfordBlue" type="heading 2">
+          Your borrow
+        </Text>
+
+        <Text color="oxfordBlue" type="body 2" as="span">
+          Balance $
+          <Text as="span" color="oxfordBlue" bold>
+            1.00
+          </Text>
+        </Text>
+      </StackLayout>
+      <ProgressBarWrapper>
+        <ProgressBar title="Borrowed" value={50} showBottomLabel />
+      </ProgressBarWrapper>
+    </BorrowColumn>
+  )
+
+  return (
+    <PageLayout {...nav}>
+      <PageLayout.Section>
+        <StackLayout center gap={1}>
+          <ColumnLayout>
+            <CardLayout>
+              <Table
+                columns={yourDepositColumns}
+                data={[
+                  {
+                    assets: 'Asset',
+                    balance: '1.00',
+                    APY: '0.50%',
+                  },
+                ]}
+                titleComponent={<DepositTitleBox />}
+              />
+            </CardLayout>
+
+            <CardLayout>
+              <Table
+                columns={yourBorrowColumns}
+                data={[
+                  {
+                    assets: 'Asset',
+                    balance: '1.00',
+                    APY: '0.50%',
+                  },
+                ]}
+                titleComponent={<BorrowTitleBox />}
+              />
+            </CardLayout>
+          </ColumnLayout>
+          <ColumnLayout>
+            <CardLayout>
+              <Table
+                columns={assetsToDepositColumns}
+                data={defaultData}
+                titleComponent="Assets to deposit"
+              />
+            </CardLayout>
+
+            <CardLayout>
+              <Table
+                columns={assetsToBorrowColumns}
+                data={defaultData}
+                titleComponent="Assets to borrow"
+              />
+            </CardLayout>
+          </ColumnLayout>
+        </StackLayout>
+      </PageLayout.Section>
+    </PageLayout>
+  )
+}
+
+const BorrowColumn = styled(ColumnLayout)`
+  position: relative;
+`
+const ProgressBarWrapper = styled.div`
```

Comment:
we need to have these stylings inside the package/ui if possible.
Meaning, I'm fine if we do something like this:

```tsx
const HeaderTitleComponent = styled.td`
  padding: 30px 20px 14px;
  font-size: 24px;
  color: ${({ theme }) => theme.tableTextColor};
  background-color: ${({ theme }) => theme.tableCellBackgroundColor};

  .c-table__header--progressbar {
     position: relative;
     
    ${ProgressBar} {
        position: absolute;
        top: -20px;
        right: 0;
        width: 50%;
      }
}
`
// and then something like this

<ColumnLayout classname="c-table__header--progressbar">
      <StackLayout gap={2.125}>
        <Text as="span" color="oxfordBlue" type="heading 2">
          Your borrow
        </Text>

        <Text color="oxfordBlue" type="body 2" as="span">
          Balance $
          <Text as="span" color="oxfordBlue" bold>
            1.00
          </Text>
        </Text>
      </StackLayout>
        <ProgressBar title="Borrowed" value={50} showBottomLabel />
    </BorrowColumn>

```

Reply:
I would actually also love to have for example 

```tsx
// Table.tsx
export const stylings = { 
// Some comment how/why/when should we use this class
tableHeaderWithProgressBar: 'c-table__header--progressbar'
 }
```

So that we always keep track of the classes we are having, and easily refactor.